spec/services/build_auction_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe BuildAuction do
+  describe '#perform' do
+    context 'building with all valid params' do
+      it 'builds the auction object' do
+        user = create(:user)
+        params = {auction: FactoryGirl.attributes_for(:auction)}
```

Comment:
I think the code climate settings we have ask for spaces between curly braces (doesn't analyze spec files but would be cool to have it "in style")